### ISSUE TITLE
NFS fails in summer-2015 when launching Starcluster clusters

### ISSUE THREAD

**Issue by paciorek** [ucberkeley/bce#63]
appears to be because summer-2015 is Ubuntu 15.04 while spring-2015 was Ubuntu 14.04.  Starcluster uses upstart to start up the NFS daemon while 15.04 uses systemd.  I'm going to try out a hack with /etc/init.d/nfs passing things through to systemd, but this is on the edge of my sysadmin skills so will also check in with Ryan. 

I should also file an issue with Starcluster.

**Comment by paciorek** [comment]
Ok, in discussion with Aaron and with campus' Amazon AWS architect it sounds like CFN clusters are a better strategy than StarCluster. Aaron and I are going to explore this further.

This also means that the post-install scripts listed on the BCE webpage are only partially operational. I will try to work through what works and what doesn't and make that clearer on the webpage. 

I believe that single-instance use of BCE on AWS will work ok for both summer-2015 and spring-2015 but will work to verify that shortly.

**Comment by paciorek** [comment]
I've confirmed that the parallel tools post-install and use of Starcluster work for spring-2015. Also now noted on the BCE webpage. 

So if we don't get CFN clusters working for the D-Lab CWG, we can make use of spring-2015 with or without Starcluster.

**Comment by aculich** [comment]
---------- Forwarded message ----------
From: Chris Paciorek paciorek@stat.berkeley.edu
Date: Fri, Oct 2, 2015 at 5:13 PM
Subject: example Starcluster usage
To: Aaron Culich aculich@berkeley.edu

In my parallel processing tutorial for biostats last spring, I used Starcluster + BCE.

see: https://github.com/berkeley-scf/parallelR-biostat-2015

The starcluster.sh file shows how I started up the cluster. Then in Sections 6.1.2, 6.2.1, 6.2.2 of parallel.pdf I show examples of running R code on the cluster, with R started via mpirun. 

chris
